core/src/main/java/io/github/wysohn/triggerreactor/core/script/interpreter/Interpreter.java

```diff
@@ -360,7 +360,7 @@ private void start(Node node) throws InterpreterException {
 
                 if (context.isStopFlag())
                     return;
-                Token valueToken =  context.popToken();
```

Comment:
Looks great! Thank you so so much.

Can I ask a favor? Since I'm going through a refactoring process, so there is a high chance of conflicts.

Would you mind removing all the unnecessary (well actually necessary for readability) modification like this one?

I think we have to come up with an unified formatting rule (maybe this: https://www.jetbrains.com/help/idea/configuring-code-style.html, but I'm open to any suggestions)

But in the meantime, please reduce the line of changes as small as possible to avoid conflicts!

Reply:
Oops, that is totally my fault.
Didn't think of the co-op so much, I will do fixing ASAP.